spec/controllers/heartbeat_controller_spec.rb

```diff
@@ -2,27 +2,70 @@
 
 describe HeartbeatController do
   render_views
-  
-  context "key restrictions" do
-    context "are enabled" do
+
+  context "with key restrictions" do
+    context "enabled" do
       it "raises an error" do
         Rails.configuration.metrics_auth_key = ""
         controller.should_receive(:reject!)
         get :healthcheck
       end
     end
-        
-    context "are disabled" do
+
+    context "disabled" do
       before :each do
         controller.stub(:reject_without_key!)
       end
 
       it "talks to sendgrid and messagelabs" do
         PrisonMailer.should_receive(:smtp_settings).and_return(address: host = double, port: port = double)
-        SendgridHelper.should_receive(:smtp_alive?).with(host, port).once
+        SendgridHelper.should_receive(:smtp_alive?).with(host, port).once.and_return(true)
         VisitorMailer.should_receive(:smtp_settings).and_return(address: host = double, port: port = double)
-        SendgridHelper.should_receive(:smtp_alive?).with(host, port).once
+        SendgridHelper.should_receive(:smtp_alive?).with(host, port).once.and_return(true)
         get :healthcheck
+
+        assert_response(:success, response.status)
```

Comment:
Unfortunately not.

Because we're on rspec 2, expectations don't map between response symbol names and numeric status codes, and I thought it nicer to use the symbol, even though it does mean we're not being quite so rspec-native. Waddya think?